contents/tutorials/react-popups.md

```diff
@@ -0,0 +1,168 @@
+---
+title: How to add popups to your React app with PostHog
+date: 2023-03-17
+author: ["ian-vanagas"]
+showTitle: true
+sidebar: Docs
+featuredImage: ../images/tutorials/banners/tutorial-3.png
+topics: ['feature flags', 'experimentation']
+---
+
+Popups are a way to highlight features in your app. This tutorial shows how to add them to a React app and control them using feature flags and JSON payloads. The use of feature flag payloads enables easy customization and control of popups.
```

Comment:
```suggestion
Popups are a way to highlight features in your app. This tutorial shows how to add them to a React app and control them using feature flags and JSON payloads. Feature flag payloads are an ideal tool for this task because they enable you to send arbitrary data, which you can use to control popups without needing to redeploy. 
```